components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-overview/reporting-overview.component.spec.ts

```diff
@@ -305,7 +305,7 @@ describe('ReportingOverviewComponent', () => {
       component.onPlatformChanged({name: 'ubuntu'});
 
       expect(router.navigate).toHaveBeenCalledWith(['/compliance', 'reports', 'nodes'],
-        {queryParams: { platform: ['ubuntu'] }});
+        {queryParams: { platform_with_version: ['ubuntu*'] }});
```

Comment:
Created new filter types of `platform_with_version` and `profile_with_version` to not break the existing API requests.